src/MagnumPlugins/UfbxImporter/UfbxImporter.conf

```diff
@@ -0,0 +1,76 @@
+depends=AnyImageImporter
+provides=FbxImporter
+provides=ObjImporter
+
+# [configuration_]
+[configuration]
+
+# Generate normals if missing.
+generateMissingNormals=false
+
+# Fail if the file is slightly malformed.
+strict=false
+
+# Disable any exporter-specific quirks in the importer.
+disableQuirks=false
+
+# Attempt to load further files referenced by this file.
+# On .obj files this would include .mtl files.
+loadExternalFiles=true
+
+# Ignore all geometry (vertices, indices) during loading.
+ignoreGeometry=false
+
+# Ignore all animation data (curves) during loading.
+ignoreAnimation=false
+
+# Ignore embedded data (such as images) during loading.
+ignoreEmbedded=false
+
+# Ignore all the actual content in the file, only loading the scene information
+# such as object relationships and names.
+ignoreAllContent=false
+
+# Maximum amount of temporary memory in bytes to use, negative for unlimited.
+maxTemporaryMemory=-1
+
+# Maximum amount of result memory in bytes to use, negative for unlimited.
+maxResultMemory=-1
```

Comment:
Maybe mention that it fails to import if the limit is exceeded? (I assume it won't choose some slower but more memory-efficient path in that case.)

Reply:
Yep it will just fail cleanly, need to remember to document that to ufbx as well!